src/app/GitUI/UserControls/OutputHistoryControl.cs

```diff
@@ -0,0 +1,14 @@
+﻿using GitCommands;
+using ResourceManager;
+
+namespace GitUI.UserControls;
+
+public partial class OutputHistoryControl : GitExtensionsControl
```

Comment:
If there are no specific reasons I'd encourage keeping all new types to be `internal` rather than `public`. Generally, only interfaces that participate in DI should be public.